rotkehlchen/tests/api/test_ethereum_transactions.py

```diff
@@ -1330,11 +1332,12 @@ def test_events_filter_params(rotkehlchen_api_server, ethereum_accounts, start_w
     tx2 = make_ethereum_transaction(tx_hash=b'2', timestamp=2)
     tx3 = make_ethereum_transaction(tx_hash=b'3', timestamp=3)
     tx4 = make_ethereum_transaction(tx_hash=b'4', timestamp=4)
-    event1 = make_ethereum_event(tx_hash=b'1', index=1, asset=A_ETH, timestamp=TimestampMS(1), location_label=ethereum_accounts[0])  # noqa: E501
+    test_contract_address = string_to_evm_address('0xc37b40ABdB939635068d3c5f13E7faF686F03B65')
```

Comment:
why not use make_evm_address from factories here? Is there any reason to have this specific address?